lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -20,6 +20,18 @@
   // the end of the text
   width: calc(min(#{$max-width}, 100%));
   max-width: max-content;
+  position: relative;
+
+  &::after {
+    border: 1px solid var(--nx-color-interactive-border-focus);
+    filter: drop-shadow(var(--nx-drop-shadow-focus));
+    content: '';
+    width: 22px;
+    height: 22px;
+    position:absolute;
+    left: -4px;
+    display: none;
+  }
```

Comment:
This is minor, and a personal thing (Ross doesn't share this view), but I prefer to see CSS attributes in alphabetical order.

Exceptions are that min/max/height, min/max/width are grouped, and top/right/bottom/left are grouped after `position`.

Reply:
> This is minor, and a personal thing (Ross doesn't share this view), but I prefer to see CSS attributes in alphabetical order.
> 
> Exceptions are that min/max/height, min/max/width are grouped, and top/right/bottom/left are grouped after `position`.

Noted. Fixed.
b711591bb80b4e5c3cae2dc00d0587dd74f80419